app/controllers/chapters_controller.rb

```diff
@@ -140,37 +140,79 @@ def chapter_3
 
     @map_2 = {
       id: 'map_2',
-      tiles_url: 'https://tiles.arcgis.com/tiles/Mj0hjvkNtV7NRhA7/arcgis/rest/services/PP_Live_Ch2_Fg5_Oct20/VectorTileServer/tile/{z}/{y}/{x}.pbf',
-      layers: [
-        {
-          id: 'over-ten-' + random_number,
-          text_large: 'Over 10%',
-          source_layers: { poly: 'Ch2_Fg5_mcat5' },
-          colour: PURPLE_SCHEME[3]
-        },
-        {
-          id: 'six-to-ten-' + random_number,
-          text_large: '6% - 10%',
-          source_layers: { poly: 'Ch2_Fg5_mcat4' },
-          colour: PURPLE_SCHEME[2]
-        },
-        {
-          id: 'three-to-six-' + random_number,
-          text_large: '3% – 6%',
-          source_layers: { poly: 'Ch2_Fg5_mcat3' },
-          colour: PURPLE_SCHEME[1]
-        },
+      csv_url: URI.join(root_url, "/file/#{CSV_CH3_MAP_PA_NATIONAL}"),
```

Comment:
You could replace this with the `get_csv_url` method you defined in ChaptersHelper (after changing that first to get the new paths)